public/app/features/plugins/admin/components/InstallControls/ExternallyManagedButton.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { getExternalManageLink } from '../../helpers';
+import { HorizontalGroup, LinkButton } from '@grafana/ui';
+
+type ExternallyManagedButtonProps = {
+  pluginId: string;
+  pluginStatus: 'install' | 'update' | 'uninstall';
```

Comment:
**Nit:** maybe `pluginStatus` is a type that we could define in a more central place

Only bringing it up as it is used in multiple places.